BookingsApi/BookingsApi.Common/Const/HearingScottishVenueNames.cs

```diff
@@ -0,0 +1,20 @@
+using System.Collections.Generic;
+
+namespace BookingsApi.Common.Const;
```

Comment:
these can probably live in BookingsApi.DAL.Helper, that's where the other constants seem to be stored

Reply:
moved